modules/services/strongswan-swanctl/param-constructors.nix

```diff
@@ -0,0 +1,161 @@
+# In the following context a parameter is an attribute set that
```

Comment:
This come seems generic enough so it can be in another PR or at least not in a folder specific to strongswan